subt/docker/robotika/ros_proxy_node.cc

```diff
@@ -52,7 +52,7 @@
 #include <assert.h>
 
 
-#define ROSBAG_SIZE_LIMIT 2000000000L  // 2GB
+#define ROSBAG_SIZE_LIMIT 3000000000L  // 2GB
```

Comment:
I would change the comment here ... also the change is a bit dangerous as they are officially guarantying only 2GB

Reply:
Done.

As long as it works, I'd like to use it. When we can get comfortably under 2GB again for the 1h run, we can switch back.